doc/interfaces/gibbs_unringing_flow.rst

```diff
@@ -0,0 +1,72 @@
+.. _gibbs_unringing_flow:
+
+===============
+Gibbs Unringing
+===============
+
+This tutorial walks through the steps to perform Gibbs unringing using DIPY.
+
+You can try this using your own data; we will be using the data in DIPY.
+You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+---------------------------
+Suppress Gibbs Oscillations
+---------------------------
+
+Magnetic Resonance (MR) images are reconstructed from the Fourier coefficients
+of acquired k-space images. Since only a finite number of Fourier coefficients
+can be acquired in practice, reconstructed MR images can be corrupted by Gibbs
+artefacts, which is manifested by intensity oscillations adjacent to edges of
+different tissue types [1]_. Although this artefact affects MR images in
+general, in the context of diffusion-weighted imaging, Gibbs oscillations
+can be magnified in derived diffusion-based estimates [1]_, [2]_.
+
+We will use the ``tissue_data`` dataset in DIPY to showcase this denoising
+method. As with any other workflow in DIPY, you can also use your own data!
```

Comment:
denoising has been signaled to be controversial or not a term globally adopted/accepted for Gibbs unringing. Not sure if this is a consequence of directly borrowing the paragraph from the denoising tutorial. It should be changed to something like:

We will use the ``tissue_data`` dataset in DIPY to showcase the ability to remove Gibbs ringing artefacts.

Reply:
Reworded in 19f6367